### PR TITLE
:white_check_mark: Add tests for snow ignore

### PR DESCRIPTION
**Pull request by daniellanner**
# Pull Request Template

## Description
SnowFS provides the ability to ignore files within the repository when executing snow commands. Ensure desired behaviour of the ignore feature and improve doumentation for edge cases by writing tests.

Fixes # (102)

## Checks

- [x] I added my name to the contributor list in **CONTRIBUTORS.md**
- [x] I understand that my work will be licensed under the given license of `SnowFS`
- [x] I am the original author, or I have authorization to submit this work.

### CODE REVIEW

File: test/11.ignore.ts [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+      if (code === 0) {
+        // if used in Visual Studio these are some debug outputs added to the output
+        std = std.replace(/Debugger attached./, '').trimLeft();
+        std = std.replace(/Waiting for the debugger to disconnect.../, '').trimRight();
+        resolve(std ?? undefined);
+      } else {
+        reject(Error(`Failed to execute ${command} ${args.join(' ')} with exit-code ${code}\n${std}`));
+      }
+    });
+  });
```
Comment (seb-mtl) [inline test/11.ignore.ts:45]: I will create a helper file for this
